### PR TITLE
manual qa documentation updated

### PR DESCRIPTION
**Pull request by mdacoca**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have created a changelog fragment 
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+> **Note:** While in review mode, all other tools will be hidden.
+
+**Review** mode screen looks like the following:
+
+![Review mode screen](/images/review_mode_screen.jpg)
+
+### Assigning reviewer
+
+> **Note**: Only project owners and maintainers can assign reviewers.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:44]: That is not exactly true.

Reviewers can be assigned by project/task owner/assignee.
And by organization maintainers of course.

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+1. Log in to the reviewer's account.
+2. On the **Controls** sidebar, click **Open and issue** (![](/images/image195.jpg)).
+3. Click on the area of the frame where the issue is occurring,
+   and the **Issue report popup** will appear.
+
+   ![Issue report window](/images/issue_report.jpg)
+
+4. In the text field of the **Issue report popup**, enter the issue description.
+5. Click **Submit**.
+6. After finishing the review, save the work.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:70]: That is not obligatory. You do not need to save the work in advance. 
Issues are created on the server immediately after submission

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+1. Right-click on the area of the frame where the issue is occurring.
+2. From the popup menu select one of the following:
+
+   - **Open an issue...**: to create new issue.
+   - **Quick issue: incorrect position**: to report incorrect position of the label.
+   - **Quick issue: incorrect attribute**: to report incorrect attribute of the label.
+   - **Quick issue...**: to open the list of issues that were reported by you before.
+
+3. After finishing the review, save the work.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:96]: The same comment about saving

File: site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+
+   ![Issue report](/images/resolve_issue.jpg)
+
+6. After all issues are fixed save work, go to the **Menu** select the **Change the job state** and
+   change state to **Complete**.
+
+   ![Change job status](/images/image197.jpg)
+
+You can easily access multiple issues created in one
+location by hovering over an issue and scrolling the mouse wheel.
```
Comment (bsekachev) [inline site/content/en/docs/manual/advanced/analytics-and-monitoring/manual-qa.md:135]: Do we need this to be dublicated?

**Comment by bsekachev** [comment]
Maybe we should mention, that review mode is available only for 2D content 
Not available for 3D point clouds

**Comment by mdacoca** [comment]
> Maybe we should mention, that review mode is available only for 2D content Not available for 3D point clouds

It is mentioned at the very beginning of the document.

**Review by bsekachev — APPROVED** [review]

**Review by azhavoro — APPROVED** [review]

**Comment by bsekachev** [comment]
We will merge it after release just in case

**Pull request merged**